src/website/utils/fields.py

```
@@ -1,182 +1,131 @@
-        logger.error(f"Validation error for file '{file.name}': {ve.message}")
-        raise
+        get_image_dimensions(file)
     except Exception as e:
-        # Catch-all for any other exceptions
-        error_msg = f"An unexpected error occurred while processing '{file.name}': {e}"
-        logger.error(error_msg)
-        raise ValidationError(error_msg)
-
-# Helper function for custom upload paths
+        logger.error(f"Invalid image file '{file.name}': {e}")
+        raise ValidationError(f"The file '{file.name}' is not a valid image.")
```

Ruff (0.8.0)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)